### PR TITLE
Rename buffergeometry_drawcalls example to _drawrange

### PR DESCRIPTION
**Pull request by fernandojsg**
This has been bothering me for a while, as I don't remember why I named the example with `drawcalls` instead of `drawrange` as the example is about the later not about draw calls :)

### CODE REVIEW

File: examples/webgl_buffergeometry_drawrange.html [hunk 1]
```diff
@@ -0,0 +1,293 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - buffergeometry - lines drawcalls</title>
```
Comment (Mugen87) [inline examples/webgl_buffergeometry_drawrange.html]: In this case you can also change the title 😊

**Comment by WestLangley** [comment]
>I don't remember why I named the example with drawcalls

We used to use the terminology "draw calls".

File: examples/webgl_buffergeometry_drawrange.html (hunk 1 above)
Comment (mrdoob) [reply examples/webgl_buffergeometry_drawrange.html]: Yep!

**Comment by mrdoob** [comment]
Shouldn't this PR also remove the `webgl_buffergeometry_drawcalls.html` file?

File: examples/webgl_buffergeometry_drawrange.html (hunk 1 above)
Comment (fernandojsg) [reply examples/webgl_buffergeometry_drawrange.html]: Nice catch, fixed :)

**Comment by fernandojsg** [comment]
> > I don't remember why I named the example with drawcalls
> 
> We used to use the terminology "draw calls".

@WestLangley ok! thanks for clarifying that makes sense :)



> Shouldn't this PR also remove the `webgl_buffergeometry_drawcalls.html` file?

@mrdoob yeah, fixed, I forgot to add the change on the rename so it appeared just as new file instead of renaming :)

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**